Dockerfile-windows.dapper

```diff
@@ -0,0 +1,57 @@
+FROM golang:1.12-windowsservercore
+SHELL ["powershell", "-NoLogo", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+RUN [Environment]::SetEnvironmentVariable('PATH', ('c:\innoextract;c:\app;{0}' -f $env:PATH), [EnvironmentVariableTarget]::Machine)
+RUN $URL = 'http://constexpr.org/innoextract/files/innoextract-1.7-windows.zip'; \
```

Comment:
I prefer to keep the download URL because it is inspired by [Drone Docker plugin Dockerfile](https://github.com/drone-plugins/drone-docker/blob/master/docker/docker/Dockerfile.windows.1809#L8-L10).